WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -95,7 +95,8 @@ public object CreateWallet(string walletName, string password)
 	{
 		var walletGenerator = new WalletGenerator(Global.WalletManager.WalletDirectories.WalletsDir, Global.Network);
 		walletGenerator.TipHeight = Global.BitcoinStore.SmartHeaderChain.TipHeight;
-		var (keyManager, mnemonic) = walletGenerator.GenerateWallet(walletName, password);
+		// TODO: Add support for multi-share wallet backup
```

Comment:
Remove the comment. It is not necessary to add the functionality here.

Reply:
In fact, after moving code from KeyManager to WalletGenerator you should undo the changes in this file.

Reply:
> Remove the comment. It is not necessary to add the functionality here.

https://github.com/WalletWasabi/WalletWasabi/pull/13800/commits/7a5e799338a02ae31903847e5dd0befc9d19c9d6

Reply:
> Remove the comment. It is not necessary to add the functionality here.

https://github.com/WalletWasabi/WalletWasabi/pull/13800/commits/7a5e799338a02ae31903847e5dd0befc9d19c9d6

Reply:
> In fact, after moving code from KeyManager to WalletGenerator you should undo the changes in this file.

https://github.com/WalletWasabi/WalletWasabi/pull/13800/commits/2371fbc0569c05145fb5c3f0e47a0cf2d10fb33c